internal/update/realtime/realtime.go

```diff
@@ -315,7 +321,7 @@ func populateStopSequences(trip *gtfs.Trip, existingStopTimes []db.ListTripStopT
 			continue
 		}
 		// If the stop sequence from the feed is valid, we use that.
-		if stopTime.StopSequence != nil && lastSequence < int64(*stopTime.StopSequence) {
+		if !reassignStopSequences && stopTime.StopSequence != nil && lastSequence < int64(*stopTime.StopSequence) {
```

Comment:
Would it be possible at the start of this function to introduce a variable

useFeedStopSequences := !reassignStopSequences

and use that here instead of !reassignStopSequences? I think this may be more readable.

This suggestion probably comes across as random, but actually I have always found the implementation of this function really tricky to follow (even though I wrote it), and feel this change might help keep cognitive load down...!